crypto/mem.c

```diff
@@ -122,12 +122,13 @@ static void parseit(void)
 }
 
 /*
- * Windows doesn't have random(), but it has rand()
+ * Windows doesn't have random() and srandom(), but it has rand() and srand().
```

Comment:
(trivial, please don't iterate just for this) should more properly be "Windows doesn't have random() or srandom()"